site/_layouts/documentation.html

```diff
@@ -268,7 +284,23 @@ <h3>Rules</h3>
            <h3>Reference</h3>
            <ul class="sidebar-nav">
              <li><a href="/versions/{{ current_version }}/user-manual.html">Commands and Options</a></li>
-             <li><a href="/versions/{{ current_version }}/skylark/build-style.html">BUILD Style Guide</a></li>
+
+	     <li>
+                <a class="sidebar-nav-heading" data-toggle="collapse"
+                    href="#build-files-menu" aria-expanded="false"
+                    aria-controls="build-files-menu">
+                  BUILD Files<span class="caret"></span>
+                </a>
+                <ul class="collapse sidebar-nav sidebar-submenu" id="build-files-menu">
+		  <li><a href="/versions/{{ current_version }}/be/functions.html">Functions</a></li>
+                  <li><a href="/versions/{{ current_version }}/be/common-definitions.html">Common Definitions</a></li>
+                  <li><a href="/versions/{{ current_version }}/be/make-variables.html">"Make" Variables</a></li>
+		  <li><a href="/versions/{{ current_version }}/skylark/build-style.html">BUILD Style Guide</a></li>
+                </ul>
+              </li>
+
```

Comment:
indentation, and good idea for a dedicated BUILD files section :)

Reply:
Done.